cernopendata/modules/fixtures/data/records/cms-tools-heavyion-examples.json

```diff
@@ -0,0 +1,116 @@
+[
+  {
+    "abstract": {
+      "description": " <p><p>This is an example of the C++ code (and its configuration in Python) needed to extract information from CMS heavy-ion data and write them to a derived dataset. An example analysis routine is provided to illustrate how to analyse data in this derived output file. These instructions are valid to work with 2010 data. Snippets of this software can be used for physics analysis.</p> <p>The examples, within the code, have in-line comments to make it more understandable and to point potential users to the original information sources. It has been put together compiling the most important pieces of already-available <a href=\"https://github.com/cms-sw/cmssw\">CMSSW software</a>.</p></p> "
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Bylinkin, Alexandr"
```

Comment:
We can add a DOI if you happen to know it. E.g.the following one? https://orcid.org/0000-0001-6286-120X

Reply:
I could not find ORCID that for him, thanks for that! We can add that and I'll ask him to confirm (also the licence) once the records are on dev